plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -599,6 +600,40 @@ private String getLongInClause(int start, int length)
         return "orderkey IN (" + longValues + ")";
     }
 
+    @Test(dataProvider = "testCaseInsensitivePredicatePushdownDataProvider")
+    public void testCaseInsensitivePredicatePushdown(String type)
```

Comment:
BCT candidate?

Or we need `BaseJdbcConnectorTest` for cases like this?

Reply:
> BCT candidate?

Sure these tests should be ported, but it is out of the scope of this PR.

> Or we need BaseJdbcConnectorTest for cases like this?

Not yet. This feature only exists in few JDBC connectors. So I think it is too specific to have generic tests.

Reply:
> Sure these tests should be ported, but it is out of the scope of this PR.

sure, let's have an issue

we also need something generic for TopN pushdows, see https://github.com/trinodb/trino/issues/7031